build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '1.0.44'
+version '1.0.45'
```

Comment:
The last tag is 1.0.45, so may be increment it to 1.0.46?

Reply:
OK, last time in the code it was 1.0.44 but I see the tag doesn't align, I'll bump to 1.0.46